datafusion/sqllogictest/test_files/window.slt

```diff
@@ -2010,13 +2010,10 @@ Projection: ARRAY_AGG(aggregate_test_100.c13) AS array_agg1
 --------TableScan: aggregate_test_100 projection=[c13]
 physical_plan
 ProjectionExec: expr=[ARRAY_AGG(aggregate_test_100.c13)@0 as array_agg1]
---AggregateExec: mode=Final, gby=[], aggr=[ARRAY_AGG(aggregate_test_100.c13)]
-----CoalescePartitionsExec
-------AggregateExec: mode=Partial, gby=[], aggr=[ARRAY_AGG(aggregate_test_100.c13)]
---------RepartitionExec: partitioning=RoundRobinBatch(2), input_partitions=1
-----------GlobalLimitExec: skip=0, fetch=1
-------------SortExec: fetch=1, expr=[c13@0 ASC NULLS LAST]
---------------CsvExec: file_groups={1 group: [[WORKSPACE_ROOT/testing/data/csv/aggregate_test_100.csv]]}, projection=[c13], has_header=true
+--AggregateExec: mode=Single, gby=[], aggr=[ARRAY_AGG(aggregate_test_100.c13)]
```

Comment:
The reason behind this is that the fetch=1 option is now available, which is a very cool feature.